{
  try {
    mAnim.resume();
  }
 catch (  junit.framework.AssertionFailedError e) {
    mFuture.setException(new RuntimeException(e));
  }
}
